packages/calcite-design-tokens/.prettierrc

```diff
@@ -0,0 +1,19 @@
+{
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "printWidth": 120,
+  "semi": true,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false,
+  "overrides": [
+    {
+      "files": "*.tsx",
+      "options": {
+        "printWidth": 100
+      }
+    }
+  ]
+}
```

Comment:
Can you remove the `.prettierrc` and `.editorconfig` files so the whole monorepo has consistent formatting?